scripts/cc-cli/package.json

```diff
@@ -7,7 +7,7 @@
     "creditcoin-cli": "dist/index.js"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "npx jest",
```

Comment:
Why use `npx`? It may not be installed everywhere. 

For `integration-tests/package.json` we use 
```
        "test": "jest --silent --runInBand",
```